scripts/python/packages/parthenon_tools/parthenon_tools/contour1d.py

```diff
@@ -0,0 +1,285 @@
+# ========================================================================================
+# Parthenon performance portable AMR framework
+# Copyright(C) 2020-2025 The Parthenon collaboration
+# Licensed under the 3-clause BSD License, see LICENSE file for details
+# =========================================================================================
+# (C) (or copyright) 2020-2025. Triad National Security, LLC. All rights reserved.
+#
+# This program was produced under U.S. Government contract 89233218CNA000001 for Los
+# Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+# for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+# in the program are reserved by Triad National Security, LLC, and the U.S. Department
+# of Energy/National Nuclear Security Administration. The Government is granted for
+# itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+# license in this material to reproduce, prepare derivative works, distribute copies to
+# the public, perform publicly and display publicly, and to permit others to do so.
+# =========================================================================================
+
+from __future__ import print_function
+
+import numpy as np
+from scipy import interpolate
+from functools import partial
+
+try:
+    from phdf import phdf
+except ModuleNotFoundError:
+    from parthenon_tools.phdf import phdf
+
+from argparse import ArgumentParser
+from multiprocessing import Pool
+import matplotlib.pyplot as plt
+
+
+parser = ArgumentParser(prog="contour1d", description="Make a spacetime contour plot")
+parser.add_argument(
+    "--components",
+    dest="tc",
+    type=int,
+    nargs="+",
+    default=None,
+    help="Components of field to plot.",
+)
+parser.add_argument(
+    "--colorbar", type=str, default=None, help="Add a colorbar with the specified label"
+)
+parser.add_argument(
+    "--logcolor", action="store_true", help="Colorbar is on a log scale"
+)
+parser.add_argument("--colormap", type=str, default="plasma", help="Colormap to use")
+parser.add_argument(
+    "--colorpoints",
+    type=float,
+    default=[],
+    nargs="+",
+    help="Points used on colormap for contours",
+)
+parser.add_argument(
+    "--workers",
+    "-w",
+    help="Number of parallel workers to use (default: 10)",
+    type=int,
+    metavar="COUNT",
+    default=10,
+)
+parser.add_argument(
+    "--savename",
+    help="Name to save file as",
+    default="plot.png",
+)
+parser.add_argument(
+    "--xlim",
+    type=float,
+    help="x bounds. Defaults to whole domain.",
+    nargs=2,
+    default=None,
+)
+parser.add_argument("--xlabel", type=str, help="Label for x axis", default="Time")
+parser.add_argument(
+    "--ylim",
+    type=float,
+    help="y bounds. Defaults to whole domain.",
+    nargs=2,
+    default=None,
+)
+parser.add_argument("--ylabel", type=str, help="Label for y axis", default="radius")
```

Comment:
Should there be a specific default value if there's there specific default field?